### PR TITLE
enhance status display

### PR DESCRIPTION
**Pull request by btry**
Use Common ITILObject statuses instead of icons (.png) provided by the plugin

image

image

### CODE REVIEW

File: css/styles.scss [hunk 1]
```diff
@@ -717,18 +717,6 @@ a.plugin_formcreator_formTile_title {
...
-   width: 265px;
-   position: absolute;
-   right: 0;
-   top: 0;
-
-
-   > div {
-      margin-bottom: 10px;
-   }
-}
```
Comment (btry) [inline css/styles.scss:730]: conflicts with cards : main area and cards on the right (my last forms) overlap

File: css/styles.scss [hunk 2]
```diff
@@ -823,38 +811,35 @@ a.plugin_formcreator_formTile_title {
 }
 
 #plugin_formcreator_lastForms {
-   .plugin_formcreator_card {
```
Comment (btry) [inline css/styles.scss:826]: specifying .plugin_formcreator_card is useless.

File: inc/form.class.php [hunk 3]
```diff
@@ -1038,19 +1038,22 @@ protected function showMyLastForms() : void {
...
             }
-               echo '<li data-itemtype="PluginFormcreatorFormanswer" data-id="' . $formAnswer['id'] . '">';
-               echo ' <a class="plugin_formcreator_'.$status.'" href="formanswer.form.php?id='.$formAnswer['id'].'">'.$formAnswer['name'].'</a>';
-               echo '<span class="plugin_formcreator_date">'.Html::convDateTime($formAnswer['request_date']).'</span>';
-               echo '</li>';
+            $status = CommonITILOBject::getStatusClass($status);
+            echo '<li data-itemtype="PluginFormcreatorFormanswer" data-id="' . $formAnswer['id'] . '">';
+            echo '<i class="'.$status.'"></i><a href="formanswer.form.php?id='.$formAnswer['id'].'">'.$formAnswer['name'].'</a>';
+            echo '<span class="plugin_formcreator_date">'.Html::convDateTime($formAnswer['request_date']).'</span>';
+            echo '</li>';
```
Comment (btry) [inline inc/form.class.php:1056]: deindent the lines : 1 level; show status with Font Awesome

File: inc/form.class.php [hunk 4]
```diff
@@ -1082,21 +1085,24 @@ protected function showMyLastForms() : void {
          echo '<div class="card-body text-center text-muted" >'.__('No form waiting for validation', 'formcreator').'</div>';
       } else {
          echo '<div class="card-body">';
-         echo '<ul>';
+         echo '<ul class="list-group">';
```
Comment (btry) [inline inc/form.class.php:1088]: class was missing; see the 1st card in showMyLastForms() (a few dozens line above)

**Review by orthagh — APPROVED** [review]

**Pull request merged**